src/components/card/card.less

```diff
@@ -34,6 +34,7 @@
 
     .text {
       flex: 1;
+      word-wrap: break-word;
```

Comment:
`break-word` may still cause some issues. Particularly in languages like german where a single "word" may go beyond the width of the card. Any objections to using `ellipsis`?